### ISSUE TITLE
blob.h runs afoul of new C++20 rules

### ISSUE THREAD

**Issue by warwick-davis** [capnproto/capnproto#1752]
https://github.com/capnproto/capnproto/blob/c17f9eace395433688b759313840aa741a45aea5/c%2B%2B/src/capnp/blob.h#L177
This line causes an error in Visual Studio 2022 >17.6 claiming that 'rewritten candidate function was excluded from overload resolution because a corresponding operator!= declared in the same scope'.

**Comment by germandiagogomez** [comment]
I am not super knowledgeable but can you try:


```
 inline bool operator!=(const char* a, const Text::Reader & b) { return a != b.asString(); } 
```

**Comment by kentonv** [comment]
Fixed in release 1.0.1.
